tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -18,6 +19,121 @@
 logger = logging.getLogger(__name__)
 
 
+# FIXME: DOESNT WORK ON REGRESSION PREDICTIONS
```

Comment:
Can we make it work for regression?

Reply:
Yes, it works for regression tasks now. I found that we need to, somehow, squeeze the onnx output.